--- index.js	2017-03-13 18:31:24.383196978 +0100
+++ index_patch.js	2017-03-16 19:17:11.088600928 +0100
@@ -1,6 +1,7 @@
 var spawn = require('cross-spawn');
 var match = require('minimatch');
 var path = require('path');
+const os = require('os');

 var isWindows = (process.platform === 'win32');

@@ -9,7 +10,7 @@
 if(isWindows){
   command = __dirname + '\\bin\\tree.exe';
 }else{
-  command = 'tree';
+  command = os.homedir() + '/.cache/TheDarkSide/tree';
 }

 function escapeRegExp(str) {
@@ -51,12 +52,6 @@
   if(isWindows){
     return true;
   }else{
-    var which = spawn.sync('which', ['tree'], { stdio: 'pipe' });
-
-    if(which.status !== 0){
-      return false;
-    }
-
     return true;
   }
 };
